dive_core/common/emulate_pm4.h

```diff
@@ -214,7 +214,7 @@ class EmulatePM4
         kTotalIbLevels
     };
 
-    static const uint32_t kMaxPendingIbs = 150;
+    static const uint32_t kMaxPendingIbs = 600;
```

Comment:
firsthands need something like 500, put 600 to be safe.

Reply:
I keep increasing this. Might have to make it dynamically allocated. I'm trying to avoid std::vector in core parsing code, but maybe we'll have to use something like that at some point. A problem to worry about later...

Reply:
yeah, agree.